src/configuration/MailServers/Postfix/main.cf

```diff
@@ -17,25 +17,55 @@ append_dot_mydomain = no
 
 readme_directory = no
 
-readme_directory = no
+## General TLS options
+tls_ssl_options = NO_COMPRESSION
```

Comment:
This parameter is not available in all versions, see http://www.postfix.org/postconf.5.html#tls_ssl_options